src/packages/client/fixtures/blog/prisma/schema.prisma

```diff
@@ -1,6 +1,6 @@
 datasource db {
-  provider = "postgresql"
-  url      = env("DB_URL")
+  provider = "sqlite"
+  url      = "file:./dev.db"
```

Comment:
Please keep this example as it is, we oftentimes need to debug Json and therefore need Postgres